src/stories/checkbox/Checkbox.stories.ts

```diff
@@ -42,6 +42,22 @@ export const Surface: Story = {
     },
 };
 
+export const Panel: Story = {
+    args: {
+        children: 'Credit/debit card',
+        kind: 'panel',
+    },
+    render: (args) => {
+        // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
I believe to avoid this "exception", and good practice is creating a wrapper:

```
const CheckboxWrapper = (args) => {
    const [checked, setChecked] = useState(false);
    return createElement(Checkbox, {
        ...args,
        checked,
        onChange: (e) => setChecked(e.target.checked),
    });
};

export const Panel: Story = {
    args: {
        children: 'Credit/debit card',
        kind: 'panel',
    },
    render: CheckboxWrapper,
};
```